pdns/pdnsutil.cc

```diff
@@ -121,7 +121,7 @@ void loadMainConfig(const std::string& configdir)
 
   // Keep this line below all ::arg().set() statements
   if (! ::arg().laxFile(configname.c_str()))
-    cerr<<"Warning: unable to read configuration file '"<<configname<<"'."<<endl;
+    cerr<<"Warning: unable to read configuration file '"<<configname<<"': "<<strerror(errno)<<endl;
```

Comment:
Is there a reason `laxFile` can return false and `errno` is not set? If that would be the case, the error message would be lying (as it uses an old `errno`).

Reply:
maybe laxFile should just return (-)errno...?